streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryTimeOrderedKeyValueBuffer.java

```diff
@@ -116,7 +118,7 @@ public StateStore build() {
 
         @Override
         public Map<String, String> logConfig() {
-            return Collections.emptyMap();
+            return logConfig;
```

Comment:
It might not be safe to return a mutable reference, I'm not sure. What about wrapping it with "unmodifiableMap"?

Reply:
Done